### DISCUSSION TITLE
Table Style

### DISCUSSION THREAD

**Discussion by Luigicube** [ShapeCrawler/ShapeCrawler#648] [Q&A] [answered]
Can we access table style as shown in the screenshot below?

![image](https://github.com/ShapeCrawler/ShapeCrawler/assets/5346257/998823cd-453f-4c2f-9036-38a89afa7752)

**Comment by AdamShakhabov** [comment, marked as answer]
No, it is not available yet.

**Comment by AdamShakhabov** [comment]
A new issue https://github.com/ShapeCrawler/ShapeCrawler/issues/655 has been created to implement this feature.
